source/amazon/services/supported-services/trusted-advisor.rst

```diff
@@ -5,97 +5,130 @@
 
-    .. thumbnail:: ../../../images/aws/aws-create-firehose-10.png
+    .. thumbnail:: ../../../images/aws/trusted-kinesis-7.png
       :align: center
-      :width: 70%
+      :width: 100%
+
+
+Amazon EventBridge configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Configure an Amazon EventBridge rule to send Trusted Advisor events to the Amazon Kinesis Data Firehose delivery stream created in the previous step.
 
-#. The following page is just a summary about the Firehose stream created, go to the bottom of the page and click on the *Create delivery stream* button:
+#. Search for ``EventBridge`` in the search bar at the top of the page or go to ``Services`` > ``Application Integration`` > ``EventBridge``:
 
-    .. thumbnail:: ../../../images/aws/aws-create-firehose-11.png
+    .. thumbnail:: ../../../images/aws/trusted-eventbridge-1.png
       :align: center
-      :width: 70%
+      :width: 100%
 
-#. Go to Services > Management Tools > CloudWatch:
+#. Click on the ``Create rule`` button:
 
-    .. thumbnail:: ../../../images/aws/aws-create-firehose-12.png
+    .. thumbnail:: ../../../images/aws/trusted-eventbridge-2.png
       :align: center
       :width: 70%
 
-#. Select *Rules* on the left menu and click on the *Create rule* button:
+#. Give an appropriated name for the EventBridge rule:
 
-    .. thumbnail:: ../../../images/aws/aws-create-firehose-13.png
+    .. thumbnail:: ../../../images/aws/trusted-eventbridge-3.png
       :align: center
-      :width: 70%
+      :width: 100%
 
-#. Select which service do you want to get logs from using the *Service name* slider, then, click on the *Add target* button and add the previously created Firehose delivery stream there. Also, create a new role to access the delivery stream:
+#. In the ``Define pattern`` section choose ``Event pattern``, then ``Pre-defined pattern by service``. Select ``AWS`` and ``Trusted Advisor`` as the ``Service provider`` and ``Service name``, respectively. Choose ``All Events`` as the desired ``Event type``:
 
-    .. thumbnail:: ../../../images/aws/aws-create-firehose-14.png
+    .. thumbnail:: ../../../images/aws/trusted-eventbridge-4.png
       :align: center
-      :width: 70%
+      :width: 100%
+
+#. Select the Firehose delivery stream created following the Kinesis steps as the ``Target`` for this EventBridge rule. Create a new role or specify an existing one for this resource if required:
+
+    .. thumbnail:: ../../../images/aws/trusted-eventbridge-5.png
+      :align: center
+      :width: 100%
 
-#. Give the rule some name and click on the *Create rule* button:
 
-    .. thumbnail:: ../../../images/aws/aws-create-firehose-15.png
+#. Scroll down and click on the ``Create rule``. The new rule will now be present in the ``Amazon EventBridge`` > ``Rules`` section, ready to be used. From now on, every time a Trusted Advisor event is sent, it will be stored in our S3 bucket. Remember to first enable the service to monitor, otherwise no data will be processed:
```

Comment:
Fix typo. Also, I suggest a couple of rewordings that I think make the paragraph sound a little better.
```suggestion
#. Scroll down and click on ``Create rule``. The new rule will now be present in the ``Amazon EventBridge`` > ``Rules`` section, ready to be used. From now on, every time a Trusted Advisor event is sent, it will be stored in the specified S3 bucket. Remember to enable the Trusted Advisor service first, otherwise no data will be processed:
```